orc8r/lib/go/metrics/metrics_export.go

```diff
@@ -78,9 +79,15 @@ func GetMetrics() ([]*prometheus_proto.MetricFamily, error) {
 	}
 	// timeStamp in milliseconds
 	timeStamp := time.Now().UnixNano() / int64(time.Millisecond)
+	// labels for metrics
+	labels := make([]*prometheus_proto.LabelPair, 0)
```

Comment:
Nit: use a nil slice, since the size of the slice will be small

Reply:
Hi Hunter, thanks for your comments. From what I've read, that change would be removing the `make()`, is that right? If that's correct please confirm, and if not, would you mind elaborating a little bit more? Sorry about that, I'm still a newbie with Go 😅

Reply:
For sure no worries. Here's the intro https://gobyexample.com/ and here's all you need to know https://go.dev/doc/effective_go

In this particular case, you can initialize a nil variable as `var labels []*prometheus_proto.LabelPair`

Reply:
Awesome, thanks!